### PR TITLE
PAAS-1023: adding README

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-1023

Short description:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+These images are all hosted on Docker Hub.
+
+For each template, docker image tags can be enabled/disabled to restrict node creation to specific versions.
+
+| Node type        | Docker image                                    |
+| ---------------- | ----------------------------------------------- |
+| Haproxy          | jelastic/haproxy:2.1.4                          |
+| Jahia Browsing   | jahia/jahiastic-jahia (from 7.2.2.0 to 8.0.0.0) |
+| Jahia Processing | jahia/jahiastic-jahia (from 7.2.2.0 to 8.0.0.0) |
+| MariaDB          | jelastic/mariadb:10.4.13                        |
```
[the remark this answers is not in the document]
Comment (adoux) [reply README.md]: So you don't trust your colleagues?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+This repository hosts all needed stuff to create a Jahia environment on Jelastic.
+
+## Infrasctructure overview
+
+A Jahia environment contains:
+- Two Haproxy nodes
+- One or several Jahia Browsing nodes
+- One Jahia Processing node
+- Either one single MariaDB node or three MariaDB nodes clusterized with Galera
```
[the remark this answers is not in the document]
Comment (adoux) [reply README.md:11]: We are not using Galera when there's only a single MariaDB node, cf mariadb mixin:
```
    - if (nodes.sqldb.length > 1):
        - installGaleraCluster
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+# Jelastic Jahia Environments
+
+This repository hosts all needed stuff to create a Jahia environment on Jelastic.
```
[the remark this answers is not in the document]
Comment (adoux) [reply README.md]: Done :rage:

File: README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+# Jelastic Jahia Environments
+
+This repository hosts all ~~stuff~~ packages, scripts & config files needed to create a Jahia environment on Jelastic.
```
Comment (kevinvandel) [inline README.md:3]: :neutral_face: 
I'm not sure a public repo is a good place for private jokes :p

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinvandel) [reply README.md]: When we add a new tag to jahia/jahiastic-jahia, we don't have to update this repo, so we won't open a PR just to update the readme :p

File: README.md (hunk 4 above)
Comment (adoux) [reply README.md:3]: I knew you were not funny, but I have to update something anyway so...

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (laurentfufu) [reply README.md]: VERY, VERY SAD COMMIT.

**Pull request merged**